supervisor/validate.py

```diff
@@ -180,10 +181,24 @@ def validate_repository(repository: str) -> str:
 SCHEMA_AUTH_CONFIG = vol.Schema({sha256: sha256})
 
 
+@dataclass
+class UserInfo:
```

Comment:
Let's only add things that we now need. Move this also into const.py and we should call it IngressSessionData.